apps/meteor/client/components/dashboards/getClosedPeriod.spec.ts

```diff
@@ -0,0 +1,85 @@
+import { getClosedPeriod } from './periods';
+
+jest.mock('moment', () => {
+	return () => jest.requireActual('moment')('2024-05-19T12:00:00.000Z');
+});
+
+it('should return the correct period range for this month', () => {
+	const monthExpectedReturn = {
+		start: new Date('5/1/2024').toISOString().split('T')[0],
+		end: new Date('5/19/2024').toISOString().split('T')[0],
+	};
+
+	expect(getClosedPeriod('month')(true).start.toISOString().split('T')[0]).toEqual(monthExpectedReturn.start);
+	expect(getClosedPeriod('month')(true).end.toISOString().split('T')[0]).toEqual(monthExpectedReturn.end);
```

Comment:
```suggestion
        const period = getClosedPeriod('month')(true);
	expect(period.start.toISOString().split('T')[0]).toEqual(monthExpectedReturn.start);
	expect(period.end.toISOString().split('T')[0]).toEqual(monthExpectedReturn.end);
```
_The same can be applied to all other test cases._